### PR TITLE
Try to generate changesets on Renovate PRs

### PR DESCRIPTION
**Pull request by AaronMoat**
This is difficult to test without merging 🙈 

Last bit to resolve https://github.com/seek-oss/skuba/issues/1493

### CODE REVIEW

File: .github/workflows/renovate-changeset.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      - name: Configure Git
+        run: |
+          git config --global user.email 34733141+seek-oss-ci@users.noreply.github.com
+          git config --global user.name 'Github changeset workflow'
+      - name: Generate changeset
+        uses: actions/github-script@v7
+        env:
+          PR_TITLE: ${{ github.event.pull_request.title }}
+        with:
+          script: |
```
Comment (zbrydon) [inline .github/workflows/renovate-changeset.yml:30]: Do actions requite this to be inline?

**Review by zbrydon — APPROVED** [review]

File: .github/workflows/renovate-changeset.yml (hunk 1 above)
Comment (AaronMoat) [reply .github/workflows/renovate-changeset.yml:30]: Not directly, but it's a bit gross either way https://github.com/actions/github-script?tab=readme-ov-file#run-a-separate-file

File: .github/workflows/renovate-changeset.yml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+            const { stdout: shortHash } = await exec.getExecOutput(
+              'git rev-parse --short HEAD',
+            );
+            const fileName = `.changeset/renovate-${shortHash.trim()}.md`;
+
+            await createChangeset(fileName, prTitle, packageNames);
+            await exec.exec('git', ['add', fileName]);
+            await exec.exec('git commit -C HEAD --amend --no-edit');
+            await exec.exec('git push --force');
```
Comment (samchungy) [inline .github/workflows/renovate-changeset.yml:87]: Not sure if we're requiring commit signing here but this doesn't look like it'll sign yeah?

Comment (AaronMoat) [reply .github/workflows/renovate-changeset.yml:87]: TBH unsure - as it's replacing the commit. I don't think we require it atm

**Comment by AaronMoat** [comment]
Going to try and likely revert 😆

**Pull request merged**